styles.css

```diff
@@ -0,0 +1,89 @@
+html {
+  font-size: 100%;
+  line-height: 1.5rem;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  min-height: 100vh;
+  max-width: 1400px;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+  display: block;
+}
+
+/* toolbar */
+.header-nav {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  padding: 25px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 58px;
+  align-items: center;
+  padding: 64px 24px;
+  position: absolute;
+  background: url(images/header-illsutration-mobile@2x.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.headline-title {
+  font-family: inherit;
+  font-size: 40px;
```

Comment:
- [ ] For better responsiveness, when adding width, height, margin, and padding to your project, always favor the use of percent-based units (vw, vh, %) over the fixed unit (px).